### DISCUSSION TITLE
直播网页全屏后抽奖弹窗不能显示

### DISCUSSION THREAD

**Discussion by zhj9709** [the1812/Bilibili-Evolved#2000] [综合讨论 / General]
![image](https://user-images.githubusercontent.com/13991774/123957939-0503c680-d9df-11eb-9f4d-66e205716ee3.png)
![image](https://user-images.githubusercontent.com/13991774/123957971-11881f00-d9df-11eb-9681-65d018e67145.png)

类似上面那样，网页全屏后，那个弹窗不能显示。网页全屏时，主播开启抽奖，也看不到弹窗，关闭脚本后没问题

**Comment by the1812** [comment]
bug 请发在 Issues 区, 后续跟进见 #2013
